### PR TITLE
BLUEVIEW DRIVER: add noise threshold option, optional no timer pings

### PR DESCRIPTION
**Pull request by kev-the-dev**
Some quick fixes, should improve performance on driver with some tunning.

### CODE REVIEW

File: drivers/mil_blueview_driver/include/blueview_ros_driver.hpp
Comment (DSsoto) [inline drivers/mil_blueview_driver/include/blueview_ros_driver.hpp]: This should be replaced with getter, setter funcs (if necessary), bad practice to have public data members.

File: drivers/mil_blueview_driver/include/blueview_wrapper.hpp
Comment (DSsoto) [inline drivers/mil_blueview_driver/include/blueview_wrapper.hpp]: You should add doc strings for these public methods, in blueview_ros_driver.hpp as well.

File: drivers/mil_blueview_driver/src/blueview_ros_driver.cpp
Comment (DSsoto) [inline drivers/mil_blueview_driver/src/blueview_ros_driver.cpp:168]:
```
Dude, get used to using function try blocks!

int main()
try
{
  do_stuff();
}
catch(this)
{
  handle_this();
}
catch(that)
{
  handle_that();
}
```

**Review by DSsoto — CHANGES_REQUESTED** [review]
Two simple requests.

File: drivers/mil_blueview_driver/src/blueview_ros_driver.cpp
Comment (kev-the-dev) [reply drivers/mil_blueview_driver/src/blueview_ros_driver.cpp:168]: Huh? This is a try catch block

**Comment by kev-the-dev** [comment]
Comments addressed but you hold on just one minute. I'm going to refactor so it's more useful to the ros community by allowing you to just untar the blueview sdk from the CD that comes with the sonar it instead of relying on our special cmake package for the sdk. refer to uf-mil/installer#11

File: drivers/mil_blueview_driver/src/blueview_ros_driver.cpp
Comment (DSsoto) [reply drivers/mil_blueview_driver/src/blueview_ros_driver.cpp:168]: You naive youngster..... oooooiiiiee

**Comment by kev-the-dev** [comment]
Note: don't merge until uf-mil/installer#12 is merged

File: drivers/mil_blueview_driver/include/blueview_ros_driver.hpp
Comment (DSsoto) [inline drivers/mil_blueview_driver/include/blueview_ros_driver.hpp]: Private data members should have a trailing underscore, according to the [ros style guide](http://wiki.ros.org/CppStyleGuide#Member_variables), especially since you want to contribute this package. this is why auto-formatting will be great.

File: drivers/mil_blueview_driver/src/blueview_ros_driver.cpp
Comment (DSsoto) [inline drivers/mil_blueview_driver/src/blueview_ros_driver.cpp]: Lol stickler comment, but I feel like we've been referencing blueview as one word. Consistency is always nice.

**Review by DSsoto — APPROVED** [review]

**Pull request merged**
